lib/src/sai_redis_route.cpp

```diff
@@ -288,3 +280,16 @@ sai_status_t sai_bulk_get_route_entry_attribute(
 
     return SAI_STATUS_NOT_IMPLEMENTED;
 }
+
+REDIS_GENERIC_QUAD_ENTRY(ROUTE_ENTRY,route_entry);
+
+const sai_route_api_t redis_route_api = {
+
+    REDIS_GENERIC_QUAD_API(route_entry)
+
+    // TODO: upstream signiture fix to SAI repo
+    (sai_bulk_create_route_entry_fn)sai_bulk_create_route_entry,
```

Comment:
why this is a cast ? is one of the params not matching ?

Reply:
Yes, const for pointer of pointer

---
In reply to: [153569444](https://github.com/Azure/sonic-sairedis/pull/264#discussion_r153569444) [](ancestors = 153569444)